host_core/lib/host_core/claims/server.ex

```diff
@@ -0,0 +1,26 @@
+defmodule HostCore.Claims.Server do
+  require Logger
+  use Gnat.Server
+
+  # Topic wasmbus.rpc.{prefix}.claims.put, .get
+  # claims fields: call_alias, issuer, name, revision, tags, version, public_key
+
+  def request(%{topic: topic, body: body}) do
+    cmd = topic |> String.split(".") |> Enum.at(4)
+    Logger.info("Received claims command (#{cmd})")
+    # PUT
+    if cmd == "get" do
+      # {:reply, claims}
+    else
```

Comment:
Just a note to implement this when we need it, is there an issue for this? (#5 perhaps)